traffic_ops/traffic_ops_golang/deliveryservice/request/comment/comments.go

```diff
@@ -186,3 +199,257 @@ WHERE id=:id RETURNING last_updated`
+	}
+
+	where, orderBy, pagination, queryValues, errs := dbhelpers.BuildWhereAndOrderByAndPagination(inf.Params, cols)
+	if len(errs) > 0 {
+		errCode = http.StatusBadRequest
+		userErr = util.JoinErrs(errs)
+		api.HandleErr(w, r, tx, errCode, userErr, nil)
+		return
+	}
+
+	if inf.Config.UseIMS {
+		runSecond, maxTime = ims.TryIfModifiedSinceQuery(inf.Tx, r.Header, queryValues, selectMaxLastUpdatedQuery(where, orderBy, pagination))
+		if !runSecond {
+			log.Debugln("IMS HIT")
+			api.AddLastModifiedHdr(w, maxTime)
+			w.WriteHeader(http.StatusNotModified)
+			return
+		}
+		log.Debugln("IMS MISS")
+	} else {
+		log.Debugln("Non IMS request")
+	}
+
+	deliveryServiceRequestComments := []tc.DeliveryServiceRequestCommentV5{}
+	query := selectQuery() + where + orderBy + pagination
+	rows, err := inf.Tx.NamedQuery(query, queryValues)
+	if err != nil {
+		api.HandleErr(w, r, tx, http.StatusInternalServerError, nil, errors.New("querying cdn locks: "+err.Error()))
+		return
+	}
+	defer rows.Close()
+
+	for rows.Next() {
+		var deliveryServiceRequestComment tc.DeliveryServiceRequestCommentV5
+		if err = rows.Scan(&deliveryServiceRequestComment.Author, &deliveryServiceRequestComment.AuthorID, &deliveryServiceRequestComment.DeliveryServiceRequestID, &deliveryServiceRequestComment.XMLID, &deliveryServiceRequestComment.ID, &deliveryServiceRequestComment.LastUpdated, &deliveryServiceRequestComment.Value); err != nil {
+			api.HandleErr(w, r, tx, http.StatusInternalServerError, nil, errors.New("scanning delivery service request comments: "+err.Error()))
+			return
+		}
+		deliveryServiceRequestComments = append(deliveryServiceRequestComments, deliveryServiceRequestComment)
+	}
+	api.WriteResp(w, r, deliveryServiceRequestComments)
+}
+
+// Validate is used to ensure that the DeliveryServiceRequestCommentV5 struct passed in to the function is valid.
+func Validate(dsrc tc.DeliveryServiceRequestCommentV5) error {
+	errs := validation.Errors{
+		"deliveryServiceRequestId": validation.Validate(dsrc.DeliveryServiceRequestID, validation.NotNil),
+		"value":                    validation.Validate(dsrc.Value, validation.NotNil),
+	}
+	return util.JoinErrs(tovalidate.ToErrors(errs))
+}
+
+// Update is used to modify an existing DeliveryServiceRequestCommentV5 in the database.
+func Update(w http.ResponseWriter, r *http.Request) {
+	var deliveryServiceRequestComment tc.DeliveryServiceRequestCommentV5
+
+	inf, userErr, sysErr, errCode := api.NewInfo(r, []string{"id"}, nil)
+	if userErr != nil || sysErr != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, errCode, userErr, sysErr)
+		return
+	}
+	defer inf.Close()
+
+	tx := inf.Tx.Tx
+	if err := json.NewDecoder(r.Body).Decode(&deliveryServiceRequestComment); err != nil {
+		api.HandleErr(w, r, tx, http.StatusBadRequest, err, nil)
+		return
+	}
+
+	if err := Validate(deliveryServiceRequestComment); err != nil {
+		api.HandleErr(w, r, tx, http.StatusBadRequest, err, nil)
+		return
+	}
+
+	var current TODeliveryServiceRequestComment
```

Comment:
**TODeliveryServiceRequestComment** struct uses **DeliveryServiceRequestCommentNullable** struct which has depreciated **TimeNoMod** in **LastUpdated** field.